lib/board.rb

```diff
@@ -0,0 +1,82 @@
+# require_relative "./player.rb"
+# require_relative "./players/human.rb"
+# require_relative "./players/computer.rb"
+# require_relative "./game.rb"
+require 'pry'
+
+class Board
+	attr_accessor :cells
+	def initialize
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+	end
+
+	def reset!
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+	end
+
+	def display
+		puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+		puts "-----------"
+		puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+		puts "-----------"
+		puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+	end
+
+	def position(user_input_string)
+		cells[user_input_string.chomp.to_i - 1] # user is entering 1-9.  posiitons are 0-8
+	end
+
+	def update(position_arg, player_object)
+		if valid_move?(position_arg)  #added to try and fix human#move spec
+			cells[position_arg.chomp.to_i - 1] = player_object.token
+		else
+			nil
+		end
+	end
+
+	def full?
+		if cells.any? {|cell| cell.strip.empty?}
+			false
+		else
+			true
+		end
```

Comment:
Prefer implicit return values of boolean statements. You can reverse (or negate) the boolean with the `!` (bang) operator.

`!cells.any? { |cell| cell.strip.empty?}`

However, Ruby generally provides a more apt iterator. `Board#full?` is asking if all elements are not empty strings. You can use logic that is symmetrical to those semantics with the [`#all?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-all-3F) iterator.

`cells.all? {|cell| !cell.strip.empty? }`

This is also more efficient in that the loop will stop after the iteration for which a cell is not empty.